python/createModels/helpFunctions.py

```diff
@@ -0,0 +1,18 @@
+def splitData(X, Y, amount):
```

Comment:
Please use `sklearn` [train_test_split](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html)
It's awesome :)